schemas/quality_performance_score.yaml

```diff
@@ -0,0 +1,50 @@
+---
+quality_performance_score:
+  anyOf:
+  - type: object
+    title: error
+    properties:
+      error:
+        type: object
+        properties:
+          message:
+            type: string
+          code:
+            type: string
+  - type: object
+    properties:
+      data:
+        type: object
+        properties:
+          translations:
+            type: array
+            items:
+              type: object
+              properties:
+                engine:
+                  type: string
+                score:
+                  type: number
+                id:
+                  type: string
+      errors:
+        type: array
+        items:
+          type: object
+          properties:
+            message:
+              type: string
+            code:
+              type: string
+            id:
+              type: string
```

Comment:
(optional) Perhaps you can add a helpful `description` here to let them know that the `id` here refers to the segment/translation id and not some error id.